views/styles.html

```diff
@@ -1,12 +1,16 @@
 {% extends "_donut.html" %}
 
-{% block pageTitle %}{{pageTitle}}{% endblock %}
-
 {% block pageHead %}
   {{ super() }}
 {% endblock %}
 
 {% block main %}
+  <section class="_style-guide--group">
```

Comment:
While I see what you're looking to do here, the Punchcard style guide isn't set up in the same way that our previous work was. Punchcard needs to be updated to do so in order to use this (please do so).

In addition, custom components should be below our base styling